### PR TITLE
Description revision for buttons

### PR DESCRIPTION
**Pull request by c-yuwei**
Description changes for less confusion:
image
image
image

The followings were tested:
- [x] 'test_PPCompareEnsembleMP_CON_FUN_CPDict_BUD'
- [x] 'test_PPCompareEnsembleMP_CON_FUN_CPDict_BUT'
- [x] 'test_PPCompareEnsemble_CPDict_BUD'
- [x] 'test_PPCompareEnsemble_CPDict_BUT'
- [x] 'test_PPCompareEnsembleMP_FUN_CPDict_BUD'
- [x] 'test_PPCompareEnsembleMP_FUN_CPDict_BUT'
- [x] 'test_PPCompareEnsemble_FUN_CPDict_BUD'
- [x] 'test_PPCompareEnsemble_FUN_CPDict_BUT'
- [x] 'test_PPAnalyzeGroupMP_G'
- [x] 'test_PPCompareGroup_CPDict_ST_MP_BUD'
- [x] 'test_PPCompareGroup_CPDict_ST_MP_BUT'
- [x] 'test_PPCompareGroup_CPDict_ST_BUD'
- [x] 'test_PPCompareGroup_CPDict_ST_BUT'
- [x] 'test_PPAnalyzeEnsemble_ME_DICT'
- [x] 'test_PPAnalyzeEnsembleMP_ME_DICT'
- [x] 'test_PPAnalyzeGroup_G'

### CODE REVIEW

File: braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m [hunk 1]
```diff
@@ -32,13 +32,6 @@
 f_adj % figure for plot adj
 f_g % figure for class graph
 
-%% ¡props!
```
Comment (c-yuwei) [inline braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m:35]: PG is not used in this element. It has to be removed for passing the test

File: braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m [hunk 2]
```diff
@@ -32,13 +32,6 @@
...
-
-%%% ¡prop!
-PG (data, item) is a plot graph.
-%%%% ¡settings!
-'PlotGraph'
-
```
Comment (c-yuwei) [inline braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m:41]: PG is not used in this element. It has to be removed for passing the test

File: braph2genesis/src/analysis/_PPAnalyzeEnsemble_ME_DICT.gen.m [hunk 3]
```diff
@@ -37,14 +37,7 @@
...
 %%%% ¡default!
-'on'
-
-%% ¡props!
-
-%%% ¡prop!
-PG (data, item) is a plot graph.
-%%%% ¡settings!
-'PlotGraph'
+'inactive'
```
Comment (c-yuwei) [inline braph2genesis/src/analysis/_PPAnalyzeEnsemble_ME_DICT.gen.m:40]: The valid value for "ENABLE" is "inactive" or "off"

File: braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m (hunk 2 above)
Comment (c-yuwei) [reply braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m:41]: @egolol please help check if this is appropriate to do this way

File: braph2genesis/src/analysis/_PPAnalyzeEnsemble_ME_DICT.gen.m (hunk 3 above)
Comment (c-yuwei) [reply braph2genesis/src/analysis/_PPAnalyzeEnsemble_ME_DICT.gen.m:40]: @egolol please help check if this is appropriate to do this way

File: braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m (hunk 2 above)
Comment (c-yuwei) [reply braph2genesis/src/analysis/_PPAnalyzeEnsembleMP_ME_DICT.gen.m:41]: checked, it is ok

File: braph2genesis/src/analysis/_PPAnalyzeEnsemble_ME_DICT.gen.m (hunk 3 above)
Comment (c-yuwei) [reply braph2genesis/src/analysis/_PPAnalyzeEnsemble_ME_DICT.gen.m:40]: checked, it is ok

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**